apps/bridge-dapp/src/containers/Layout/Layout.tsx

```diff
@@ -1,12 +1,34 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
 import { Header } from '../../components/Header';
-import { Footer } from '@webb-tools/webb-ui-components/components';
+import { Banner, Footer } from '@webb-tools/webb-ui-components/components';
 
 export const Layout: FC<{ children?: React.ReactNode }> = ({ children }) => {
+  const [showBanner, setShowBanner] = useState(true);
+
+  const onCloseHandler = () => {
+    setShowBanner(false);
+  };
+
   return (
     <div className="min-w-full min-h-full">
       <Header />
 
+      {showBanner && (
+        <Banner
+          onClose={onCloseHandler}
+          dappName="stats"
+          buttonText="Report Bug"
+          bannerText="Hubble Bridge is in beta version."
+          buttonProps={{
+            onClick: () =>
+              window.open(
+                'https://github.com/webb-tools/webb-dapp/issues/new/choose',
+                '_blank'
+              ),
```

Comment:
Consider using `href` and `target` here to generate the `<a></a>` instead of `≤button></button>` here.